### DISCUSSION TITLE
Work in Progress

### DISCUSSION THREAD

**Discussion by 14yannick** [14yannick/HCPBridgeMqtt#1] [Ideas]
I whould like to discuss what should we work on next.

I have a pir sensor on my desk which I could integrate. please give my your inputs. @Gifford47 @fisch55

**Comment by Gifford47** [comment]
in order to use a pir it should be configurable in the configuration.h like the bme sensor. i would prefer a interrupt service routine on core 0 in order to avoid a interrupt of the communication on core1. in the isr we have to check the state of the gpio and set a flag for the main loop. with a pir we have also a new topic (or we use the json in the state topic) with new payload. the same for the HA-discovery message,

but beside from the pir, i think we have to investigate why the door doesen´t respond sometimes to open/close commands, see https://github.com/Gifford47/HCPBridgeMqtt/issues/11
or is it only at my garage? :)
